### DISCUSSION TITLE
It's a real amazing.

### DISCUSSION THREAD

**Discussion by Wenrh2004** [community/community#67326] [GitHub Education]
### Hi everyone,

I feel like I'm being played around like a fool and the fact that no one will get back to you! I don't feel any sincerity!Because no one cares about us and it doesn't matter if we complete the certification or not. We probably shouldn't be participating in this activity, we don't have an equal right to this right that is rightfully ours, and in this case I feel like my legal rights have been taken away from me and taken a toll on me, and that I'm doing it out of a sense of being a beggar, and not an equal participant. I think this is extremely unopen source and fairness.
<img width="1304" alt="截屏2023-09-17 00 49 32" src="https://github.com/community/community/assets/122615371/f4f32303-7258-40ad-99b4-1e34e5da31ac">

**Comment by lumstu1024** [comment]
cn啊,国人是这样的,滥用的比较厉害,你应该想想淘宝咸鱼有多少卖包的,你要是edu,别人拿着你学校的邮箱疯狂注册,咋不能风控呢,建议中国区实行学信网(中国学生的唯一凭证)或者对接支付宝实名,不然中国滥用问题永远也得不到解决

**Reply by Wenrh2004** [reply]
确实，也是一个非常寒心的地方，但是我真正不满的地方并不是严格的审核，我支持严格的审核，也厌恶所谓的这种售卖的行为，我不满足于GitHub当今这种我有问题但是解决不了的现状，在GitHub我很难通过相关途径回复我的合法权益或者寻求问题的解决方法不是吗？如果我在Jeb或者是Apple挂工单，我会非常迅速的得到回复，而且他们会耐心的引导我解决问题，我并没有遇到过无人答复的情况。难道这不正是GitHub所欠缺的东西嘛？

**Comment by lumstu1024** [comment]
发支持工单啊,教育工单不会理你的,支持工单基本上班就会理你

**Reply by Wenrh2004** [reply]
好的，我去尝试一下，谢谢您

**Comment by samus-aran** [comment]
Hi @Wenrh2004 

This message can be an indication of many things that only the Support team can answer and resolve. So please do contact [Account or billing issues](https://support.github.com/request) Support and they will explain the details of this message.

We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
